infra/runtests.py

```diff
@@ -25,7 +25,8 @@ def green(self, s): return s
     ".*",
     "lab*.full.py",
     "wbetools.py",
-    "browser.trace"
+    "browser.trace",
+    "outline*.txt",
```

Comment:
Not needed for this PR

Reply:
It's helpful because I have the outlines generated on my computer & the tests fail if there are not-accounted-for files.